projects/step-frontend/src/lib/modules/timeseries/dashboard/time-series-dashboard.component.html

```diff
@@ -1,20 +1,41 @@
 <ng-container *ngIf="performanceViewSettings">
-  <step-ts-filter-bar
-    *ngIf="context"
-    [context]="context"
-    class="mat-card"
-    [performanceViewSettings]="performanceViewSettings"
-    [defaultFilterOptions]="settings.filterOptions"
-    [initialFilters]="contextualFilterItems"
-    (onFiltersChange)="handleFiltersChange($event)"
-    (onFilterSettingsChange)="handleFilteringChange($event)"
-    (onGroupingChange)="handleGroupingChange($event)"
-  >
-  </step-ts-filter-bar>
+  <div>
+    <div class="filter-bars-container" [style.gap]="compareModeEnabled ? '12px' : '0'">
```

Comment:
I prefer not to have styles in the templates. I would add a class "compare-mode-enabled" that is inserted when it is and put 
```
div.filter-bars-container.compare-mode-enabled {
gap: 1.2rem;
}
```
into the scss.